java/src/jmri/jmrix/loconet/LnPacketizer.java

```diff
@@ -407,6 +408,322 @@ public void run() {
         }
     }
 
+    /*
+     * Alternate Packetizer(s) start here. They can be invoked by the use of properties on the command line.
```

Comment:
It would be better to create a new class for this "Alternate Packetizer" otherwise, you have two different behaviors described by the same object.

Also, There should be a parameter in the preferences to configure this.  I can see this being something you might want to enable on a per-connection basis.  Using a command line parameter forces this for all LocoNet connections.

Reply:
The thing I like most about playing (I'm retired, I don't work) on JMRI is the new and exciting challenges. I've got as far as a pr3/pr2/locobufferII configuration option labeled "toppings" with a choice of "Pepperoni" or "Cheese". And if "jmri.jmrix.loconet.pepperoni" existed, and was a packetizer  this would work. 
I will get it done, might be a while. But I will have fun.